src/app/clients/ActionBar.tsx

```diff
@@ -109,18 +118,40 @@ const ActionBar: React.FC<Props> = ({ selected, data }) => {
                 onSubmit={submitStatus}
                 errorText={modalError}
```

Comment:
Do we not need submitAction above to be done? Or is that to be implemented later?

Reply:
Removed!